# HG changeset patch
# Parent  f3edec018825871eb7b80631f3c4411a5e349b7c
Ask for the remote before checking for pending changes in release.py

diff --git a/dist/release.py.in b/dist/release.py.in
--- a/dist/release.py.in
+++ b/dist/release.py.in
@@ -364,11 +364,11 @@
         "commit", "-m", "Back to development, {}".format(next_version)
     ])
 
-    subprocess.check_call([HG_EXECUTABLE, "-R", LIGHTSD_SOURCE_DIR, "out"])
+    remote = click.prompt("Confirm the remote destination", default="default")
+    subprocess.check_call([
+        HG_EXECUTABLE, "-R", LIGHTSD_SOURCE_DIR, "out", remote
+    ])
     if click.confirm("Are you ready to push those commit?"):
-        remote = click.prompt(
-            "Confirm the remote destination", default="default",
-        )
         subprocess.check_call([
             HG_EXECUTABLE, "-R", LIGHTSD_SOURCE_DIR, "push", remote
         ])
